includes/uploader.js

```diff
@@ -121,17 +135,64 @@ function check_file_content(isValidFile) {
 
         reader.readAsText(file);
     }
+
 }
 
 
 
-// Check file content 
+function check_columns_names(isValidContent) {
+    if (isValidContent) {
+        // Define the names accepted for the columns
+        const columns_names_reference_list = [

```

Comment:
We're using camelCase everywhere else, so shouldn't this name be `columnNamesReferenceList` (or a shorter alternative such as e.g. `expectedColumnNames`)?